### PR TITLE
fix(security): fix security issues closes #1

### LINKED ISSUE #1: Security issue
Hi,

I'm a member of the Node.js Security WG and we received a report regarding a security issue with this module. We tried inviting the author by e-mail but received no response so I'm opening this issue and inviting anyone with commit and npm publish rights to collaborate with us on a fix.

### PR DESCRIPTION
**Pull request by kbshl**
Hei,

trying to close #1 with this PR. 

For the main code, `shelljs` was replaced with `execa` to use exec file behavior.

Also, there was a need to fix some linter errors reported by the last `xo` package. Due to that most of the code was moved to es6.

Currently, I don't know which node version ranges should be supported, maybe we need to transpile the code then.

### CODE REVIEW

**Review by stevemao — APPROVED** [review]
Thanks @kbshl LGTM

**Comment by stevemao** [comment]
CC @lirantal @caioluders

File: package.json [hunk 1]
```diff
@@ -25,8 +25,9 @@
     "git"
   ],
   "dependencies": {
+    "execa": "^1.0.0",
     "meow": "^3.6.0",
-    "shelljs": "^0.7.3"
+    "shelljs": "^0.8.3"
```
Comment (lirantal) [inline package.json:30]: we can probably remove shelljs dependency right?
also from the test file

**Comment by lirantal** [comment]
Left a small comment but other than that looks ok. Things like the prettier config and lockfile config should be reviewed by the package author. 

Thanks @kbshl 👏

**Comment by armano2** [comment]
@stevemao is there any progress on this?

**Pull request closed without merging**